dotnet/src/extensions/Azure/Handlers/HttpHandler/GXHttpAzureContextAccessor.cs

```diff
@@ -164,6 +167,104 @@ public override void Abort()
 			//throw new NotImplementedException();
 		}
 	}
+	internal class GxAzureResponseHeaders : IHeaderDictionary
+	{
+		HeaderDictionary m_headers;
+		HttpResponseData m_httpResponseData;
+		internal GxAzureResponseHeaders(HttpResponseData httpResponseData)
+		{
+			m_headers = new HeaderDictionary();
+			foreach (var header in httpResponseData.Headers)
+			{
+				string[] values = new Microsoft.Extensions.Primitives.StringValues(header.Value.Select(val => val).ToArray());
+				m_headers.Add(header.Key, values);
+			}
+			m_httpResponseData = httpResponseData;
+		}
+
+		public StringValues this[string key]
+		{
+			get
+			{
+				return m_headers[key];
+			}
+			set
+			{
+				m_httpResponseData.Headers.Add(key, value.AsEnumerable());
+				m_headers[key] = value;
+			}
+		}
+
+		public long? ContentLength { get { return m_headers.ContentLength; } set {; } }
+
+		public ICollection<string> Keys { get { return m_headers.Keys; } }
+		public ICollection<StringValues> Values { get { return m_headers.Values; } }
+
+		public int Count { get { return m_headers.Count; } }
+
+		public bool IsReadOnly { get { return m_headers.IsReadOnly; } }
+
+		public void Add(string key, StringValues value)
+		{
+			m_httpResponseData.Headers.Add(key, value.AsEnumerable());
```

Comment:
If the Key already exists, what would happen? Overwrite? Crash or make it duplicate? 

Example:
- Response.AddHeader("MyHeader", "value")
- Response.AddHeader("MyHeader", "value2")

Reply:
Several problems were fixed, not only the problema about the header. One problem was related to the path of the binaries.
I'll check what happens with duplicate headers..

Reply:
It doesn't crash. If you have:
Response.AddHeader("MyHeader", "value")
Response.AddHeader("MyHeader", "value2")
The result is (value, value2)